### DISCUSSION TITLE
北京航空航天大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#14] [University]
> 回答来自群友，我仅负责搬运

北航有两个校区：
- 本部在海淀区学院路，有新北区、南区、大运村（在校外）三个宿舍区；
- 沙河校区在昌平区高教园南三街，有一个宿舍区。不同专业和年级的校区和宿舍不同，详情请咨询相关专业的学长。

**宿舍是上床下桌吗**
所有宿舍都是，但是不同宿舍柜子数量大小不一样

**教室和宿舍有没有空调**
都有

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
新北区和大运村有，沙河和南区没有

**有早自习、晚自习吗**
没有

**有晨跑吗**
没有

**每学期跑步打卡的要求是多少公里，可以骑车吗**
每个学期刷 48 次体能锻炼跑道，有的学院可以用跑步代替；还是比较好混过去的

**寒暑假放多久，每年小学期有多长**
寒假一个月，暑假两个月多一点，部分专业年级有小学期（一般是一两周）

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，疫情期间本部需要去校门取（南区 500m，北区 100m），沙河一直需要去校门取（200m）。大运村应该可以送到楼下。

**学校附近有地铁站吗**
本部：西土城、知春路；沙河：沙河站（2km）

**宿舍楼有洗衣机吗**
基本都有，在单独的洗衣房或者宿舍每层的盥洗室

**校园网怎么样**
有线网千兆+，无线网 50M+（有的地方会好一点），0.5cny/GiB

**每天断电断网吗，几点开始断**
从 2017 年开始应该就不断了

**食堂价格贵吗，会吃出异物吗**
根据内容和精致程度不同在 7 -- 18 元之间，中位数 10 元，不同食堂不太一样；异物基本没发现过

**洗澡热水供应时间**
有独立卫浴的基本是全天，集体浴室的是中午（具体时间不明）和 18--22 时

**校园内可以骑电瓶车吗，电池在哪能充电**
可以，但是一般用自行车

其他信息：
- 本部校内快递要去新主楼取，比较远；大运村快递在楼下取；沙河快递在东门外取。
- 沙河校区周围非常荒凉，请做好心理准备。门口下午和晚上会有小商贩聚集。

**Comment by lynzrand** [comment]
> 大一能带电脑吗

学校没有统一限制

> 学校里面用什么卡，饭堂怎样消费

校园卡，刷卡消费，可以通过自助机和支付宝充钱

> 学校会给学生发银行卡吗

似乎要自己办

> 学校的超市怎么样

东西比较全，基本可以满足生活要求

> 学校里面的共享单车数目与种类如何

沙河不清楚（好久没去了），学院路很多

> 现阶段学校的门禁情况如何

打过疫苗的可以直接刷卡进出，否则需要申请

> 宿舍晚上查寝吗，封寝吗，晚归能回去吗

有的学院偶尔会查，部分宿舍有晚归限制（不会堵门不让进，但是辅导员会看到）

**Reply by VOIDMalkuth** [reply]
> 学校会给学生发银行卡吗

啊，我们好像是自己办的

**Comment by GrapeLemonade** [comment]
关于地铁站：22 年初昌平线南延工程完成后学院路校区东北角会增加一个学院桥站

**Comment by falg010** [comment]
补充一下，现在沙河快递在快递站取，离宿舍不算很远。东门没有小商贩聚集了。浴室开放时间11:30-13:30 16:30-24:00

**Comment by Jing2582563005** [comment]
23年4月回答：疫情已经结束了！学校现在处于全面开放状态，基本上不设门禁，亲友需要预约来学校不过秒通过，两校区之间有地铁直达了很方便！
